Configurations/CommonBase.xcconfig

```diff
@@ -25,8 +25,8 @@
 #include? "<DEVELOPER_DIR>/AppleInternal/XcodeConfig/PlatformSupport.xcconfig"
 #include "Sanitizers.xcconfig"
 #include "SDKVariant.xcconfig"
-#include "WebKitProjectPaths.xcconfig"
 #include "WebKitTargetConditionals.xcconfig"
+#include "WebKitProjectPaths.xcconfig"
```

Comment:
Why is this change needed? The includes used to be alphabetically sorted, and someone will eventually sort them again unless there is a comment explaining why that shouldn't be done.

Reply:
It’s needed so WK_MACOS_1400 can be used in WebKitProjectPaths.xcconfig. I’ll add a comment.

Reply:
Can `WebKitProjectPaths.xcconfig` include `WebKitTargetConditionals.xcconfig` as a better fix?